app/views/finders/_facet_tags.html.erb

```diff
@@ -1,28 +1,26 @@
-<div class="js-facet-tag-wrapper">
-  <% if local_assigns[:applied_filters] %>
-    <div class="facet-tags" data-module="track-click">
-      <% local_assigns[:applied_filters].each do |applied_filter| %>
-        <div class="facet-tags__group">
-          <% applied_filter.each do |filter| %>
-            <div class="facet-tags__wrapper">
-              <p class="facet-tags__preposition"><%= filter[:preposition] %></p>
-              <div class="facet-tag">
-                <p class="facet-tag__text"><%= filter[:text] %></p>
-                <button type="button"
-                  class="facet-tag__remove"
-                  aria-label="Remove filter <%= filter[:text] %>"
-                  data-module="remove-filter-link"
-                  data-track-label="<%= filter[:data_track_label] %>"
-                  data-facet="<%= filter[:data_facet] %>"
-                  data-value="<%= filter[:data_value] %>"
-                  data-name="<%= filter[:data_name] %>">&#x2715;</button>
-              </div>
+<% if local_assigns[:applied_filters] %>
```

Comment:
What was `js-facet-tag-wrapper`? and why was it removed?

Reply:
It was a class used as a hook by the JS for updating content dynamically. I've replaced it with an ID to make it consistent with the other hooks.